pkg/test/echo/common/model.go

```diff
@@ -18,6 +18,7 @@ import "istio.io/istio/pkg/config/protocol"
 
 // TLSSettings defines TLS configuration for Echo server
 type TLSSettings struct {
+	// System cert pool would be used if RootCert field empty.
```

Comment:
Change the comment to `RootCert supplies the root certificate to use inside the echo server, will use the default system cert pool if this is empty.`